frontend/src/components/common/EditStudent.vue

```diff
@@ -768,17 +767,17 @@ export default {
         return '';
       }
     },
-    getAssignedPen(assignedPen, studentPen){
-      if (!assignedPen) {
+    getAssignedPen(assignedPen, studentPen, penMatchResult){
+      if (!assignedPen && !penMatchResult) {
         return 'waiting on fixes';
       } else if (assignedPen === studentPen) {
         return assignedPen;
       } else {
         return 'under review';
       }
     },
-    getAssignedPenTooltip(assignedPen, studentPen){
-      if (!assignedPen) {
+    getAssignedPenTooltip(assignedPen, studentPen, penMatchResult){
+      if (!assignedPen && !penMatchResult) {
         return 'The submitted student details have errors or incomplete information. Confirm the submitted student name and date of birth.';
       } else if (assignedPen === studentPen) {
         return 'Differences between the Assigned PEN and Submitted PEN indicate an existing student file has been matched to the submitted details. The Assigned PEN will be used to prevent duplication.';
```

Comment:
This message is questionable; if the assigned PEN and submitted PEN match, I don't get the start of the sentence.